### PR TITLE
ST-731 - Added new mega menu

### PR DESCRIPTION
**Pull request by maheshwaghmare**
### Description
Implemented the new mega menu.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/mega-menu/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+	let [toggle, setToggle] = useState( false );
+
+	return (
+		<div className={`stc-mega-menu-wrap ${ toggle ? 'stc-mega-menu-show' : '' }`}>
+
+			<div className='stc-mega-menu-toggle' onClick={()=>{
+				setToggle( ! toggle );
+			}}>
+				<svg width="16" height="16" viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
```
Comment (vrundakansara) [inline src/mega-menu/index.js]: Can we move this icon to icons.js?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/mega-menu/index.js (hunk 1 above)
Comment (maheshwaghmare) [reply src/mega-menu/index.js]: Done.

**Review by vrundakansara — APPROVED** [review]

**Comment by maheshwaghmare** [comment]
@Nikschavan The PR [https://github.com/brainstormforce/astra-pro-sites/pull/601](https://github.com/brainstormforce/astra-pro-sites/pull/601) needs to merge with this PR.

**Review by Nikschavan — APPROVED** [review]

**Comment by maheshwaghmare** [comment]
Resolving the conflict.

**Review by Nikschavan — APPROVED** [review]

**Review by amithire1201 — APPROVED** [review]

**Pull request merged**